js/searchForm.js

```diff
@@ -0,0 +1,39 @@
+class FormElement {
+    constructor(tag, className, className2, selector, text = "") {
+        this.tag = tag;
+        this.className = className;
+        this.className2 = className2;
+        this.selector = selector;
+        this.text = text;
+        this.elem = this.createElement(tag);
+        this.addClass(className, className2)
+        this.write(text)
+        this.appendTo(selector)
+    }
+    createElement(tag) {
+        const elem = document.createElement(tag);
+        return elem;
+    }
+
+    addClass(...classNames) {
+        this.elem.classList.add(...classNames);
+    }
+
+    write(text) {
+        this.elem.innerHTML = text;
+    }
+
+    appendTo(selector) {
+        selector.append(this.elem);
+    }
+}
+
+class SearchForm {
```

Comment:
in addition to the creation of the form, this class should contain all the other logic of it - fetching the data from the server, deboucing etc...